spec/acceptance/00_meta_spec.rb

```diff
@@ -1,29 +1,16 @@
-require 'spec_helper_acceptance'
-
-# Here we put the more basic fundamental tests, ultra obvious stuff.
-describe 'puppet' do
-  it "should have an lsbdistdescription fact" do
-    expect(fact('lsbdistdescription')).to match(/(centos|ubuntu|debian|suse)/i)
-  end
-
-  desired_version = PUPPET_VERSION[/(\d+\.\d+)/] unless puppet_enterprise?
-  desired_version = "Puppet Enterprise #{PE_VERSION[/(\d+\.\d+)/]}" if puppet_enterprise?
+# frozen_string_literal: true
 
-  it "should be version: #{desired_version}.x" do
-    actual_version = shell('puppet --version').stdout.chomp
-    expect(actual_version).to contain(desired_version)
-  end
-end
+require 'spec_helper_acceptance'
 
 describe 'logstash module' do
-  it 'should be available' do
+  it 'is available' do
     shell(
       "ls #{default['distmoduledir']}/logstash/metadata.json",
       acceptable_exit_codes: 0
     )
   end
 
-  it 'should be parsable' do
+  it 'is parsable' do
```

Comment:
Why do we have this? We already have this on the host system before we even start acceptance tests.